NotificationService-client/src/App.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {CoherenceHeader} from "@cseo/controls";
+import MailHistory from "./components/mailHistory";
+import {signOut,getToken} from "./auth/authProvider";
+import {config} from "./configuration/config";
+import { useEffect,useState } from "react";
+function App() {
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  useEffect(()=>{
+   getToken().then(()=>{
+     if(sessionStorage.getItem('msal.idtoken')!==null) 
+      setIsLoggedIn(true);
+   });
+  },[]); 
+  return (<><div style={{left:"-10000px",top:"auto",width:"1px",height:"1px",overflow:"hidden"}}>
+    <a href="#main" target="#">Skip to main</a></div> 
```

Comment:
This line is not required with the current header. It's requirement is to have a `<main id="main"></main>` wrapping tag

Reply:
thanks, done